lib/util/template/index.spec.ts

```diff
@@ -123,6 +123,12 @@ describe('util/template/index', () => {
     expect(JSON.parse(output)).toEqual({});
   });
 
+  it('to Object passing illegal number of elements', () => {
+    const userTemplate = "{{{ toJSON (toObject 'foo') }}}";
+    const outputFunc = () => template.compile(userTemplate, {});
+    expect(outputFunc).toThrow();
+  });
+
```

Comment:
Added test to increase coverage.
https://github.com/renovatebot/renovate/runs/28493167812

https://github.com/renovatebot/renovate/blob/48263f8db2b9e800f2de83fce11fce276992f81e/lib/util/template/index.ts#L35-L37